internal/mempool/ids.go

```diff
@@ -44,6 +50,7 @@ func (ids *IDs) Reclaim(peerID types.NodeID) {
 	if ok {
 		delete(ids.activeIDs, removedID)
 		delete(ids.peerMap, peerID)
+		ids.nextID = uint16(tmmath.MinInt(int(ids.nextID), int(removedID)))
```

Comment:
With all the type conversion here, I don't think the Min function really helps readability in this case. 

I'd be in favour of dropping the import and just writing it out:
```suggestion
		if removedID < ids.nextID {
			ids.nextID = removedID
		}
```

It's only one extra line. 🙂

Reply:
sure.